cdisc_rule_tester/models/rule.py

```diff
@@ -0,0 +1,223 @@
+import yaml
```

Comment:
I don't think this is necessary anymore either because we are using the model from the engine